### DISCUSSION TITLE
feature request - push notifications for repository releases

### DISCUSSION THREAD

**Discussion by jdearden1** [community/community#67593] [Mobile] [answered]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.128.0 (108490092)
Device: iPhone 6S (iOS 15.7.8)
</details>
I don't know whether or not this has been suggested yet, but I would love to get push notifications whenever a repository I'm following creates a release.

**Comment by stevepopovich** [comment, marked as answer]
Hi @jdearden1, thanks for writing in. We are looking into adding this is future versions. I added your feedback to our internal tracking.
